charts/ingresslibchart/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: ingress-library
```

Comment:
Quick comment on naming consistency.

Can the name be `ingress` or `ingresslib`? And can the chart directory match the name instead of `ingresslibchart`?

Reply:
0914d3a9